databuilder/query_language.py

```diff
@@ -319,6 +319,9 @@ def __add__(self, other):
         else:
             return NotImplemented
 
+    def __radd__(self, other):
+        return self.__add__(other)
+
```

Comment:
_Maybe_ worth a comment on `__sub__()` pointing out the `__rsub__()` would be senseless?